.github/workflows/add-lang.yml

```diff
@@ -34,11 +36,11 @@ jobs:
 # Check if language has been added previously.
       - name: Check if language is already present
         run: |
-          if [ -d "_translator-files/po/${{ env.new_lang }}" ] ; then
-            echo Error: The language [${{ env.new_lang }}] seems to already exist.
+          if [ -d "_translator-files/po/${{ env.lang_code }}" ] ; then
+            echo Error: The language [${{ env.lang_code }}] seems to already exist.
             exit 1
           else
-            echo Language: [${{ env.new_lang }}] not detected. Proceeding to add [${{ env.new_lang }}].
+            echo Language: [${{ env.lang_code }}] not detected. Proceeding to add [${{ env.lang_code }}].
```

Comment:
This should be made a bit more explanatory: Language: ... not detected doesn't explain really what this means. Maybe say something like language ... seems to be new as no Language: ... was found. Proceeding to add...